doc/leak_tracking/DETECT.md

```diff
@@ -65,7 +88,7 @@ objects (See [concepts](CONCEPTS.md) for details).
 
 However, the good news is:
 
-1. Most disposable Flutter Framework classes include instrumentation.
+1. Disposables in Flutter Framework are instrumented.
 If how your Flutter app manages widgets results in leaks,
 Flutter will catch them.
```

Comment:
should this be "leak tracker" will catch them instead of "Flutter" will catch them?

Reply:
yes!